streaming/src/main/scala/org/apache/spark/streaming/Checkpoint.scala

```diff
@@ -67,12 +67,12 @@ object Checkpoint extends Logging {
   val REGEX = (PREFIX + """([\d]+)([\w\.]*)""").r
 
   /** Get the checkpoint file for the given checkpoint time */
-  def checkpointFile(checkpointDir: String, checkpointTime: Time) = {
+  def checkpointFile(checkpointDir: String, checkpointTime: Time): Path = {
```

Comment:
(i.e. yes - "public" here doesn't mean "Spark's public API", but "public" in the sense of java/scala interfaces)